Vagrantfile

```diff
@@ -123,6 +123,18 @@ Vagrant.configure("2") do |config|
 		config.vm.box = "bento/ubuntu-20.04"
 	end
 
+	# The Parallels Provider uses a different naming scheme.
+	config.vm.provider :parallels do |_v, override|
```

Comment:
I actually downloaded the trial of Parallels yesterday to see how much Chassis supports it but the trial version didn't come with the CLI tools so thanks for making a start on this!

Reply:
you need Parallels Pro or Business plus the `vagrant-parallels` plugin

Reply:
Yeah I saw that and didn't wanna fork out the money for it 😂🙈